src/main/java/core/basesyntax/GeometricFigureSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeometricFigureSupplier {
+    private Random random = new Random();
+
+    public GeometricFigure getRandomFigure() {
```

Comment:
The method 'getRandomFigure' is misleading because it does not return a geometric figure instance but rather an enum constant representing a type of figure. Consider renaming the method to 'getRandomFigureType' to clearly represent its functionality.